docs/scheduled-transactions/revised-spec.md

```diff
@@ -0,0 +1,157 @@
+
+The schedule entity type is managed by four new HAPI operations,
+  1. The `ScheduleCreate` transaction creates a new schedule (possibly with a non-empty list of signing keys).
+  2. The `ScheduleSign` transaction adds one or more Ed25519 keys to a schedule's list of affirmed signers.
+  3. The `ScheduleDelete` transaction removes a schedule from the network state.
+  4. The `GetScheduleInfo` query gets the current state of a schedule.
+
+It is important to understand that the bytes of a scheduled transaction are never 
+directly signed by an Ed25519 key. Only `ScheduleCreate` or `ScheduleSign` bytes
+are ever signed, in the ordinary way. 
+
+## The `ScheduleCreate` transaction
+  
+The `ScheduleCreateTransactionBody` message is to be,
+  
+```  
+message ScheduleCreateTransactionBody {  
+  SchedulableTransactionBody scheduledTransactionBody = 1; // The scheduled transaction; its TransactionID must not be set
+  string memo = 2; // An optional memo with a UTF-8 encoding of no more than 100 bytes
+  Key adminKey = 3; // An optional Hedera key which can be used to sign a `ScheduleDelete` and remove the schedule
+  bool executeImmediatelyOnValidSignatures = 4; // May the scheduled transaction be executed as soon as its schedule has enough signing keys? MUST be `true`.
+  AccountID scheduledPayerAccountID = 5; // The id of the account to be charged the service fee for the scheduled transaction at the consensus time that it executes (if ever).
+}  
+```  
+
+The new `SchedulableTransactionBody` message is a strict subset of the `TransactionBody` message which omits the
+top-level `TransactionID`, `nodeAccountID`, and `transactionValidDuration` fields; and does not allow the `ScheduleCreateTransactionBody`
+and `ScheduleSignTransactionBody` messages in its `data` element.
+
+In release `0.13.0`, the fourth field has incomplete support, as stated in the comments above. That is, 
+ - Only schedules that permit immediate execution of their scheduled transaction given enough signing keys may be created. 
+
+As with all other entity types, a schedule remains in network state until it expires, even if its scheduled 
+transaction has already been executed.
+
+### Paying for scheduled transactions
+
+If the `ScheduleCreate` gives a `scheduledPayerAccountID`, the network will charge this payer the service fee 
+for the scheduled transaction at the consensus time that it executes (if ever). If no such payer is specified, 
+the network will charge the payer of the originating `ScheduleCreate`.
+
+### Receipts and duplicate creations
+
+When a `ScheduleCreate` resolves to `SUCCESS`, its receipt includes a new `ScheduleID` field
+with the id of the created schedule. Its receipt _also_ includes a new 
+`scheduledTransactionID` field with the `TransactionID` that can be used to query for the 
+record of the scheduled transaction's execution. 
+
+This `scheduledTransactionID` will be the `TransactionID` of the `ScheduleCreate`, 
```

Comment:
Will this `scheduledTransactionID` also be populated during duplicate creation scenario? Seems like that is where it would be most useful. If so, can this be mentioned in that section.

Reply:
Duplicate creation will get a "redirect" to a `ScheduleID` which contains the `scheduledTransactionID`; so they can find it easily. Not a strong preference here tho.

Reply:
Added the `scheduledTransactionID` and updated this doc.